apps/storefront/app/bloggen/_components/PostLayout/PostLayout.tsx

```diff
@@ -54,8 +49,8 @@ function PostLayout({
         <main id='main' className={classes.main}>
           <div className={classes.intro}>
             <Heading level={1}>{heading}</Heading>
-            <Ingress className={classes.ingress}>{ingress}</Ingress>
-            <Paragraph size='sm' className={classes.meta}>
+            <Paragraph className={classes.ingress}>{ingress}</Paragraph>
+            <Paragraph variant='long' size='sm' className={classes.meta}>
```

Comment:
I dont understand - should the first Paragraph here be long, and not the second? ☺️

Reply:
good cath, I put the long variant in the wrong place